src/taipy/core/_entity/_labelled.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+from typing import Optional
+
+
+class _Labelled:
+    __LABEL_SEPARATOR = ">"
+
+    def get_label(self) -> str:
+        if explicit_label := self._get_explicit_label():
+            return explicit_label
+        return self._generate_label()
```

Comment:
I think this method can be:
```python
def get_label(self) -> str:
        return self._get_explicit_label() or self._generate_label()
```
But its just a style preference, nothing wrong with the current implementation.